x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/utils.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FilterOptions } from './types';
+
+/**
+ * Convert rules filter options object to KQL query
+ *
+ * @param filterOptions desired filters (e.g. filter/sortField/sortOrder)
+ *
+ * @returns KQL string
+ */
+export const convertRulesFilterToKQL = (filterOptions: FilterOptions): string => {
+  const showCustomRuleFilter = filterOptions.showCustomRules
+    ? [`alert.attributes.tags: "__internal_immutable:false"`]
```

Comment:
fwiw, there is a constant that can be used here:
https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/constants.ts#L176

I think the original code missed that constant, but it's better we use it when we can.

Reply:
Sure, np